core/src/ast/function.rs

```diff
@@ -33,6 +33,10 @@ pub enum Function {
     },
     Ceil(Expr),
     Concat(Vec<Expr>),
+    IfNull {
+        expr: Expr,
+        expr2: Expr,
+    },
```

Comment:
How about using `then` than `expr2`?
Then... if `expr` is null, then we use `then`.
```rust
IfNull {
  expr: Expr,
  then: Expr,
}
```

ref.
https://github.com/gluesql/gluesql/blob/c716a72f31f6bc94d2f019c0c4e2e136c2d3bbdb/core/src/ast/expr.rs#L58-L62

Reply:
in plan/expr/function.rs:  Exprs::Double([expr, expr2].into_iter()), wants expr and expr2.   We could create a separate category that has expr, then, etc.  But it probably isn't worth the effort.  Is there any other way to use then?

Reply:
ok.  should we delay the eval until we need the value in the variable then or should we go ahead and evaluate then before?